docker/openemr/5.0.2/Dockerfile

```diff
@@ -40,6 +40,8 @@ RUN apk add --no-cache git build-base libffi-dev python-dev \
     && pip install -e /opt/certbot/acme -e /opt/certbot \
     && mkdir -p /etc/ssl/certs /etc/ssl/private \
     && apk del --no-cache git build-base libffi-dev python-dev
+
+RUN openssl req -x509 -newkey rsa:4096 -keyout /etc/ssl/private/selfsigned.key.pem -out /etc/ssl/certs/selfsigned.cert.pem -days 1065 -nodes -subj "/C=xx/ST=x/L=x/O=x/OU=x/CN=localhost"
```

Comment:
Don't hardcode this here. Instead, use a shell script that checks for the presence of the certificate before generating it. Same effect but safer.

Reply:
Will do

Reply:
I think Let's Encrypt might have been down at one point and so I just put that in temporarily?

Reply:
You said 'will do' but didn't -- this is /probably/ minor and we could let it go, but I want to check that was your intent.